packages/react-components/react-aria/src/activedescendant/useActiveDescendant.ts

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { isHTMLElement } from '@fluentui/react-utilities';
+import { useOptionWalker } from './useOptionWalker';
+import { ActiveDescendantOptions } from './types';
+import { ACTIVEDESCENDANT_ATTRIBUTE } from './constants';
+
+export function useActiveDescendant<TActiveParentElement extends HTMLElement, TListboxElement extends HTMLElement>(
+  options: ActiveDescendantOptions,
+) {
+  const { imperativeRef, matchOption } = options;
+  const activeParentRef = React.useRef<TActiveParentElement>(null);
+  const { listboxRef, optionWalker } = useOptionWalker<TListboxElement>({ matchOption });
+  const getActiveDescendant = () => {
+    return listboxRef.current?.querySelector(`[${ACTIVEDESCENDANT_ATTRIBUTE}]`);
```

Comment:
```suggestion
   return listboxRef.current?.querySelector<HTMLElement>(`[${ACTIVEDESCENDANT_ATTRIBUTE}]`);
```

Is there a change that we will not get `HTMLElement`? Then we can eliminate some checks.